recipes/lodepng/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <vector>
+#include <cstdint>
+#include <cassert>
+#include <fstream>
+
+#include "lodepng.h"
+
+int main()
+{
+    std::ifstream stream("bees.png", std::ios::in | std::ios::binary);
+    std::vector<uint8_t> data((std::istreambuf_iterator<char>(stream)), std::istreambuf_iterator<char>());
+
+    std::vector<uint8_t> decoded;
+    unsigned width = 0, height = 0;
+    unsigned error = lodepng::decode(decoded, width, height, data);
+    assert(static_cast<bool>(error) == false);
```

Comment:
```suggestion
```

The idea for test_package is checking the generated package, not the library functionality.